### ISSUE TITLE
Pull Request Map

### ISSUE THREAD

**Issue by jasonkliu** [YaleSTC/shifts#282]
This is a map of all current pull requests and their status, cc @mnquintana @caseywatts 
When pull requests are linked together with &rarr;, merging the rightmost should merge all preceding ones.

:thumbsup: - reviewed, ready to merge
:thumbsdown: - reviewed, merge problems
:hand: - needs review
### #171 Payforms Printed :hand:

Old pull request from July 2013. 
### #235 Report show &rarr; #239 unscheduled :thumbsup: &rarr; #244 location :thumbsup: &rarr; #268 loc groups :thumbsup:

235 fixes the popup menu and tons of Rails 3.2 issues - possibly needs review :hand:
239 is a [single commit](https://github.com/YaleSTC/shifts/compare/229_report_show...238_unscheduled?w=1) which fixes the start unscheduled shift button.
242 restores location routes etc.
268 looks to be build straight off of 244.
&rarr; BRANCH OFF 235 &rarr; #241 shift report emails  :thumbsup:
241 is a [single commit](https://github.com/YaleSTC/shifts/compare/229_report_show...215_shift-report-emails?w=1) which adds gem `daemons`. :exclamation: Why don't we just add this to 239...?
&rarr; BRANCH OFF 235 &rarr; #272 active tasks :thumbsup: 
272 is 3 commits which branch specifically off of 235.  
### #236 Fake Location Email Task &rarr; #286  :thumbsup:

Fake email addresses for the demo site. Ready to merge, rebased into rails-3.2
### #234 Asset pipeline stylesheets &rarr; #254 superfish. :thumbsup:

Fixes the asset pipeline and restores the look of the headers.
### #257 stickies &rarr; #260 announcements :thumbsup:

No idea how to merge this because it's built off both 235 report show and 254 superfish.
260 is built straight off 257 :hand: . 
&rarr; #262 data objects :thumbsup:  &rarr; #270 data types  :thumbsup: 
Basically it's also built off 257 (252_superfish is merged) but the branches diverge at 684df45c3296c8d9054cc4645ed91132a9180232.
270 is built straight off of 262, except for 262 has an extra [commit](203b78c677eae2c4392a8d755dd97c3c7155b952) at the end which 270 doesn't have. 
&rarr; #264 links :thumbsup: 
This is built off of 257 also but diverges really early.

All of the pull requests I created are unrelated to those above and should merge without conflicts.
